unity-renderer/Assets/DCLPlugins/UUIDEventComponentsPlugin/UUIDComponent/InteractionHoverCanvas/InteractionHoverCanvasController.cs

```diff
@@ -122,4 +126,21 @@ void UpdateCanvas()
         if (canvas.enabled != newValue)
             canvas.enabled = newValue;
     }
-}
\ No newline at end of file
+
+    private void Update()
+    {
+        if (Utils.IsCursorLocked)
+        {
+            anchor.anchoredPosition = defaultAnchorOffset;
+            return;
+        }
+
+        RectTransformUtility.ScreenPointToLocalPointInRectangle(
+            canvas.transform as RectTransform,
```

Comment:
I always prefer to cast directly `()` considering the function can't really work with null

Reply:
Done!